### ISSUE TITLE
`/api/v1/welcome` should indicate it actually is a talk recording server

### ISSUE THREAD

**Issue by nickvergessen** [nextcloud/nextcloud-talk-recording#8]
Just happened in the chat:

- Someone had the HPB server also configured as recording server
- The `/api/v1/welcome` endpoint exists on both, so the API worked
- It showed the HPB version and only by that I randomly managed to identify it as HPB

We will now add an "invert check" and basically fail the server when the `X-Spreed-Signaling-Features` header is returned.
But it would be better if a positive `X-Nextcloud-Talk-Recording-Features` would be introduced, also allowing to check Talk if the service is compatible with what Talk expects from the recording server.
Basically like a capability list, feel inspired by https://github.com/nextcloud/spreed/blob/3ce5c7bdaafeb8f4747f7bc05e60ebd33ada192a/lib/Signaling/Manager.php#L52-L55
